### PR TITLE
Ticket #9452 Add Gandi LiveDNS DynDNS client.

### PR DESCRIPTION
**Pull request by evantill**
- [X] Redmine Issue: https://redmine.pfsense.org/issues/9452
- [X] Ready for review
- [X] Tested on my pfsense

### CODE REVIEW

File: src/etc/inc/services.inc [hunk 1]
```diff
@@ -24,8 +24,8 @@
...
+define('DYNDNS_PROVIDER_VALUES', 'all-inkl azure azurev6 citynetwork cloudflare cloudflare-v6 cloudns custom custom-v6 digitalocean dnsexit dnsimple dnsmadeeasy dnsomatic dreamhost dreamhost-v6 duiadns duiadns-v6 dyndns dyndns-custom dyndns-static dyns easydns eurodns freedns freedns-v6 glesys gandi-livedns godaddy godaddy-v6 googledomains gratisdns he-net he-net-v6 he-net-tunnelbroker hover loopia namecheap noip noip-free ods opendns ovh-dynhost route53 route53-v6 selfhost spdyn spdyn-v6 zoneedit');
+define('DYNDNS_PROVIDER_DESCRIPTIONS', 'All-Inkl.com,Azure DNS,Azure DNS (v6),City Network,Cloudflare,Cloudflare (v6),ClouDNS,Custom,Custom (v6),DigitalOcean,DNSexit,DNSimple,DNS Made Easy,DNS-O-Matic,DreamHost,Dreamhost (v6),DuiaDns.net,DuiaDns.net (v6),DynDNS (dynamic),DynDNS (custom),DynDNS (static),DyNS,easyDNS,Euro Dns,freeDNS,freeDNS (v6),GleSYS,Gandi LiveDNS, GoDaddy,GoDaddy (v6),Google Domains,GratisDNS,HE.net,HE.net (v6),HE.net Tunnelbroker,Hover,Loopia,Namecheap,No-IP,No-IP (free),ODS.org,OpenDNS,OVH DynHOST,Route 53,Route 53 (v6),SelfHost,SPDYN,SPDYN (v6),ZoneEdit');
```
Comment (jim-p) [inline src/etc/inc/services.inc]: You have an extra space here:
`GleSYS,Gandi LiveDNS, GoDaddy,`
Should be
`GleSYS,Gandi LiveDNS,GoDaddy,`
(no space before `GoDaddy`)

**Comment by evantill** [comment]
@jim-p IS is there anything I can do to help on this PR ?

**Comment by jim-p** [comment]
Nothing more you need to do, we'll get it merged into 2.5.0 next time we go over the PRs

**Comment by rbgarga** [comment]
Please rebase your fork and resolve conflicts

**Comment by evantill** [comment]
not yet tested on my side. I will test my usecase and report back here.

**Comment by evantill** [comment]
can not test on my pfsense (version 2.4.4_3)

**Comment by rbgarga** [comment]
@evantill you can test it using a pfSense 2.5.0 snapshot.  We don't have a Gandi account to validate it so we need to hear from you it's working fine before merge

**Comment by evantill** [comment]
I need to install a 2.5.0 snapshot and apply my PR as a patch is this correct ?

**Comment by evantill** [comment]
I have problems testing it.

- I have download and installed `pfSense-CE-2.5.0-DEVELOPMENT-amd64-20190530-0821.iso.gz` on a virtual machine.
- Next I have tried to install this PR as a patch using this url `https://patch-diff.githubusercontent.com/raw/pfsense/pfsense/pull/4061.patch`

But testing the patch result in an error:  `Patch can NOT be applied cleanly (detail)`

Note: testing my initial commit on it using `https://github.com/evantill/pfsense/commit/4e85f63f1e2115c20f5d08c5e4f753aa9a8ae343.patch` will result in a successfull test `Patch can be applied cleanly (detail)`

Both seems unexpected results IMHO...

**Comment by evantill** [comment]
testing problem solved after upgrading pfSense-kernel-pfSense from 2.5.0.a.20190530.0821 to 2.5.0.a.20190605.1313

**Comment by evantill** [comment]
tested on `2.5.0.a.20190605.1313`

screenshot

**Pull request merged**